packages/orchestration/test/exos/cosmos-orchestration-account.test.ts

```diff
@@ -260,7 +260,8 @@ test('CosmosOrchestrationAccount - transfer', async t => {
 
   t.log("transfer doesn't support ERTP brands yet. see #9211");
   await t.throwsAsync(E(account).transfer(ist.make(10n), mockDestination), {
-    message: 'Brands not currently supported.',
+    // TODO #9211 register asset before transfer
+    message: 'No denomination for brand [object Alleged: IST brand]',
```

Comment:
Seems like we're just down to one todo in two places - `// TODO #9211 register asset before transfer` 👏 

Can you help me understand this better? We do `E(bankManager).addAsset('uist',...)` in common setup, but `ChainHub` isn't aware about `agoricNames` outside of what it's told?

Reply:
that's right.

I made more progress on this PR after that TODO. Now I think this PR should close #9211 and all the TODOs should be removed or replaced with a new issue.